src/lint.js

```diff
@@ -4,6 +4,8 @@ const CLIEngine = require('eslint').CLIEngine
 const path = require('path')
 const formatter = CLIEngine.getFormatter()
 const userConfig = require('./config/user')
+const promisify = require('util').promisify
+const glob = promisify(require('glob'))
```

Comment:
can you pls change to https://www.npmjs.com/package/globby which already has a promise api and is based on https://github.com/mrmlnc/fast-glob which is faster than glob

Reply:
Done.